src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Style Guide
+description: Style Guide for ethereum.org
+lang: en
+sidebar: true
+---
+
+# Ethereum.org Style Guide {#style-guide}
+
+Content on [Etheruem.org](/) is crowdsourced and primarily written by our incredible contributors. This style guide aims to standardize certain aspects of contributing content to make the contribution process smoother.
```

Comment:
Somewhere (perhaps the intro section) I think we should link to the design principles. In general I think this is a useful guide to explain the "tone/vibe" we're trying to achieve: https://ethereum.org/en/contributing/design-principles/